dll/win32/shell32/lang/zh-CN.rc

```diff
@@ -457,12 +457,12 @@ STYLE DS_SHELLFONT | WS_POPUP | WS_CAPTION
 CAPTION "文件类型"
 FONT 9, "宋体", 0, 0, 0x0
 BEGIN
-    LTEXT "已注册的文件类型(&T):", -1, 7, 10, 180, 10
+    LTEXT "已注册的文件类型(&T)：", -1, 7, 10, 180, 10
     CONTROL "", IDC_FILETYPES_LISTVIEW, "SysListView32", LVS_REPORT | LVS_SINGLESEL | LVS_SHAREIMAGELISTS | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 7, 20, 249, 115
     PUSHBUTTON "新建(&N)", IDC_FILETYPES_NEW, 140, 140, 55, 14, WS_TABSTOP
     PUSHBUTTON "删除(&D)", IDC_FILETYPES_DELETE, 200, 140, 55, 14, WS_TABSTOP
-    GROUPBOX "'%S' 扩展的详细信息", IDC_FILETYPES_DETAILS_GROUPBOX, 7, 170, 249, 70
-    LTEXT "打开方式:", -1, 12, 180, 40, 10
+    GROUPBOX "“%S”扩展的详细信息", IDC_FILETYPES_DETAILS_GROUPBOX, 7, 170, 249, 70
```

Comment:
The quoted string shouldn't be space-separated with the rest of the string? 🤔

Reply:
I searched the web for example sentences, and I see Simplified Chinese users usually don't space-separated the quote.🤔

BTW,
```suggestion
    GROUPBOX "“%s”扩展的详细信息", IDC_FILETYPES_DETAILS_GROUPBOX, 7, 170, 249, 70
```
"%s" is accidentally capitalised.